scripts.js

```diff
@@ -202,36 +207,64 @@
 
     firstTry = !firstTry
   }
-
+  function search (nameKey, myArray) {
+    for (var i = 0; i < myArray.length; i++) {
+      if (myArray[i].name === nameKey) {
+        return myArray[i]
+      }
+    }
+    return false
+  }
   function setNameToSay () {
+    totalSyllables = syllable(name)
     if (!hasSpeechSupport) {
       return
     }
     nameToSay = new window.SpeechSynthesisUtterance(name)
     nameToSay.pitch = 0.8
     nameToSay.rate = 1.25
     nameToSay.lang = 'en-US'
+    var voice
+    // forgive me for i have sinned
+    if (search('Microsoft David Desktop - English (United States)', window.speechSynthesis.getVoices()) !== false) { // windows
+      voice = search('Microsoft David Desktop - English (United States)', window.speechSynthesis.getVoices())
+    } else if (search('Fred (en-US)', window.speechSynthesis.getVoices()) !== false) { // ios
```

Comment:
You could move the `window.speechSynthesis.getVoices()` into a variable and reference it to avoid duplication.